File: src/main/java/com/smarthome/server/entity/EducationProgram.java
package com.smarthome.server.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class EducationProgram {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String course;

    private String cycle;

    private String degree;

    private String type;

    private String title;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getCourse() {
        return course;
    }

    public void setCourse(String course) {
        this.course = course;
    }

    public String getCycle() {
        return cycle;
    }

    public void setCycle(String cycle) {
        this.cycle = cycle;
    }

    public String getDegree() {
        return degree;
    }

    public void setDegree(String degree) {
        this.degree = degree;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: src/main/client/src/app/admin/education-program.ts
export class EducationProgram {

  private _id: number;
  private _course: string;
  private _cycle: string;
  private _degree: string;
  private _type: string;
  private _title: string

  get id(): number {
    return this._id;
  }

  get course(): string {
    return this._course;
  }

  get cycle(): string {
    return this._cycle;
  }

  get degree(): string {
    return this._degree;
  }

  get type(): string {
    return this._type;
  }

  get title(): string {
    return this._title;
  }

  set id(value: number) {
    this._id = value;
  }

  set course(value: string) {
    this._course = value;
  }

  set cycle(value: string) {
    this._cycle = value;
  }

  set degree(value: string) {
    this._degree = value;
  }

  set type(value: string) {
    this._type = value;
  }

  set title(value: string) {
    this._title = value;
  }
}
